custom_components/tuya_local/devices/mamnv_br151_vacuum_and_mop.yaml

```diff
@@ -0,0 +1,163 @@
+          value: random
+    - id: 4
+      type: string
+      name: direction_control
+      optional: true
+      mapping:
+        - dps_val: forward
+          value: forward
+        - dps_val: backward
+          value: reverse
+        - dps_val: turn_left
+          value: left
+        - dps_val: turnn_right
+          value: right
+        - dps_val: stop
+          value: stop
+    - id: 5
+      type: string
+      name: status
+      mapping:
+        - dps_val: "standby"
+          value: standby
+        - dps_val: "smart_clean"
+          value: cleaning
+        - dps_val: "wall_clean"
+          value: cleaning
+        - dps_val: "spot_clean"
+          value: cleaning
+        - dps_val: "room_clean"
+          value: cleaning
+        - dps_val: "goto_charge"
+          value: returning
+        - dps_val: "charging"
+          value: charging
+        - dps_val: "charge_done"
+          value: charged
+        - dps_val: "cleaning"
+          value: cleaning
+        - dps_val: "sleep"
+          value: sleep
+        - dps_val: "in_trouble"
+          value: error
+    - id: 14
+      type: string
+      name: fan_speed
+      mapping:
+        - dps_val: strong
+          value: High
+        - dps_val: normal
+          value: Medium
+    - id: 18
+      type: bitfield
+      name: error
+      mapping:
+        - dps_val: 0
+        - dps_val: 1
+          value: wheel_overload
+        - dps_val: 2
+          value: side_brush_entanglement
+        - dps_val: 3
+          value: brush_abnormality
+        - dps_val: 4
+          value: dustbin_not_installed
+        - dps_val: 5
+          value: left_collision_stuck
+        - dps_val: 6
+          value: right_collision_stuck
+        - dps_val: 7
+          value: machine_off_ground
+        - dps_val: 8
+          value: front_drop_trigger
+        - dps_val: 9
+          value: left_drop_trigger
+        - dps_val: 10
+          value: right_drop_trigger
+        - dps_val: 11
+          value: fan_abnormality
+        - dps_val: 12
+          value: water_tank_not_installed
+        - dps_val: 13
+          value: magnetic_field_abnormality
+        - dps_val: 14
+          value: slippage_abnormality
+secondary_entities:
+  - entity: sensor
+    class: battery
+    dps:
+      - id: 6
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+  - entity: sensor
+    name: Cleaning time
+    category: diagnostic
+    class: duration
+    icon: "mdi:clock-outline"
+    dps:
+      - id: 17
+        type: integer
+        name: sensor
+        unit: min
+  - entity: select
+    name: Water control
+    category: config
+    icon: "mdi:cup-water"
+    dps:
+      - id: 20
+        name: option
+        type: string
+        mapping:
+          - dps_val: "closed"
+            value: "Off"
+          - dps_val: "low"
+            value: Low
+          - dps_val: "high"
+            value: Strong
+  - entity: select
```

Comment:
Hi,
I know, but I can't see all options in robot card. 
So I create this to can set up commands easily.

https://ibb.co/fQ8VKpx
https://ibb.co/KhmZNRK

Take a look at the pictures 

Let me know if I'm doing something wrong, because I'm new in this home assistant universe.

Reply:
I would suggest adding buttons to the UI to call the vacuum.send_command service for any additional commands you use frequently enough to want in Home Assistant, rather than adding an additional select entity. Some custom vacuum cards may allow additional commands to be defined in the card alongside the standard activate / pause spot clean, locate and return to dock commands.